Source/Redemption/Private/BaseLocalizationWidget.cpp

```diff
@@ -34,4 +35,41 @@ void UBaseLocalizationWidget::SetButtonTint(UButton* Button, FSlateColor NewColo
 	NewSlateBrush.TintColor = NewColor;
 	Style.SetNormal(NewSlateBrush);
 	Button->SetStyle(Style);
+}
+
+void UBaseLocalizationWidget::SetCheckBoxTint(class UCheckBox* Box, FSlateColor NewColor)
```

Comment:
Possible fix would be to call SynchronizePRoperties() on the box at the end?

Reply:
I tried, i does not. Nandish gave me an idea to change the image instead of the tint I will help him with that later